Models/Website/Quest.php

```
@@ -116,6 +116,31 @@ public function getNpcs(bool $loadIfNotLoaded = false): ?array
 
+    /**
+     * Creates a new quest in the database
+     * @param string $name The display name of the quest
+     * @return int The ID of the newly created quest
+     * @throws \PDOException If the query fails (e.g. duplicate degenerated name)
+     */
+    public static function create(string $name): int
+    {
+        $degeneratedName = self::degenerateName($name);
+        $db = new Db('Website/DbInfo.ini');
+        return $db->executeQuery('INSERT INTO quest (name, degenerated_name) VALUES (?, ?);', array($name, $degeneratedName), true);
```

PHPStan (2.1.51)

[error] Method VoicesOfWynn\Models\Website\Quest::create() should return int but returns bool|string.


(return.type)